net-p2p/airdcpp-webclient/airdcpp-webclient-2.1.0.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_{4,5,6}} pypy{,3} )
+
+inherit cmake-utils user python-any-r1
+
+DESCRIPTION="Cross-platform Direct Connect client"
+HOMEPAGE="https://airdcpp-web.github.io/"
+SRC_URI="https://github.com/airdcpp-web/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="GPL-2+"
+SLOT="0"
+IUSE="nat-pmp +tbb +webui"
+
+RDEPEND="
+	app-arch/bzip2
+	dev-cpp/websocketpp
+	dev-libs/boost:=
+	dev-libs/geoip
+	dev-libs/leveldb:=
+	dev-libs/openssl:0=[-bindist]
+	net-libs/miniupnpc:=
+	sys-libs/zlib
+	virtual/libiconv
+	nat-pmp? ( net-libs/libnatpmp:= )
+	tbb? ( dev-cpp/tbb:= )
+"
+DEPEND="
+	virtual/pkgconfig
+	${PYTHON_DEPS}
+	${RDEPEND}
+"
+PDEPEND="webui? ( www-apps/airdcpp-webui )"
+
+# https://github.com/airdcpp-web/airdcpp-webclient/pull/248
+PATCHES=( "${FILESDIR}/${P}-disable-automagic.patch" )
```

Comment:
While this is a good thing to push upstream, it's not strictly required for Gentoo. You can use instead:

    -DCMAKE_DISABLE_FIND_PACKAGE_LibNatPmp=$(usex !nat-pmp)
    -DCMAKE_DISABLE_FIND_PACKAGE_TBB=$(usex !tbb)